Firmware/MotorControl/gcode_input.c

```diff
@@ -0,0 +1,63 @@
+#include "grbl.h"
+#include "gcode_input.h"
+
+void consume_output() {
+  // Gets the current block. Returns NULL if buffer empty
```

Comment:
Codestyle: indent with four spaces

Reply:
Done

Reply:
Don't worry about it too much.  Once it's in VSCode, right-click -> Format Document will format it to the style we're using in ODrive (Google with slight modifications).